README.md

```diff
@@ -289,6 +289,14 @@ Options:
 
           [env: OVERWRITE_FILES=]
 
+  -R, --rename-duplicate
+          Enable renaming files during file upload if duplicate exists
+
+          The renaming will occur by adding numerical suffix to the filename before the final extension. For example file.txt will be uploaded as file-1.txt, the number will
+          be increased untill a available slot is found.
+
+          [env: RENAME_DUPLICATE_FILES=]
```

Comment:
This needs to be updated as it's not synced with the text in the `--help` output.